maintainers/maintainer-list.nix

```diff
@@ -4549,6 +4549,16 @@
       fingerprint = "4749 0887 CF3B 85A1 6355  C671 78C7 DD40 DF23 FB16";
     }];
   };
+  dpc = {
```

Comment:
> Looks like you also forgot to add yourself as a maintainer of the package.

Oh, I left it in by mistake because I'm waiting for https://github.com/NixOS/nixpkgs/pull/251689 and didn't want to create a commit confict ...

But I actually did not intend to add myself to maintainer of mold itself. If being an interested user enough, I can, but it wasn't my initial intention. :D